ci-operator/templates/openshift/installer/cluster-launch-installer-remote-libvirt-e2e.yaml

```diff
@@ -685,102 +722,102 @@ objects:
     - ip: "${BASTION_SVC_IP}"
       hostnames:
       - "${REMOTE_LIBVIRT_HOSTNAME}"
-      - "alertmanager-main-openshift-monitoring.apps.${NAMESPACE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-0-0"
-      - "api.${NAMESPACE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-0-0"
-      - "canary-openshift-ingress-canary.apps.${NAMESPACE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-0-0"
-      - "console-openshift-console.apps.${NAMESPACE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-0-0"
-      - "downloads-openshift-console.apps.${NAMESPACE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-0-0"
-      - "grafana-openshift-monitoring.apps.${NAMESPACE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-0-0"
-      - "oauth-openshift.apps.${NAMESPACE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-0-0"
-      - "prometheus-k8s-openshift-monitoring.apps.${NAMESPACE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-0-0"
+      - "alertmanager-main-openshift-monitoring.apps.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-0-0"
```

Comment:
Seems like you could use `INSTANCE_PREFIX` here?  Might want to rename it to `CLUSTER_NAME` for consistency with the `setup` container...

Reply:
Sadly, no. :(

Instance pre-fix is available for the teardown container, whereas this are variable's passed in directly into the yaml openshift template. The end result is that the template doesn't know these values unless they're passed in at the job level (either in the configuration OR ci-operator handles them.)

The one thing that would probably be helpful would be to remove the INSTANCE_PREFIX variable entirely since it's not used anywhere, but it's there for historical purposes in case there was an assumption made in the teardown logic (which was mostly copied over from the x86 template.)

Reply:
Aren't we inside the `teardown` container here?  Ah, I see, we're configuring `hostAliases` on the pod or some such here, not doing something inside the `teardown` container script.  Makes sense as it stands then; sorry for the noise.